lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -367,5 +367,27 @@ def self.add_quotes(value)
         value.start_with?('"') && value.end_with?('"')
       value
     end # add_quotes
+
+    # extract value of property
+    def self.extract_value(prop, prefix=nil, match_method)
```

Comment:
will move the match_method to 1st arg because pop and prefix should be together.